docs/api/hardhat/hardhat-zksync-verify.md

```diff
@@ -32,30 +35,30 @@ Import the plugin in the `hardhat.config.ts` file:
-  },
-  zkTestnet: {
-    url: "https://testnet.era.zksync.dev", // URL of the zkSync network RPC
-    ethNetwork: "goerli", // URL of the Ethereum Web3 RPC, or the identifier of the network (e.g. `mainnet` or `goerli`)
-    zksync: true,
-    // Verification endpoint for Goerli
-    verifyURL: 'https://zksync2-testnet-explorer.zksync.dev/contract_verification'
-  }
+    goerli: {
+      url: "https://goerli.infura.io/v3/<API_KEY>" // The Ethereum Web3 RPC URL (optional).
+    },
+    zkTestnet: {
+      url: "https://testnet.era.zksync.dev", // The testnet RPC URL of zkSync Era network.
+      ethNetwork: "goerli", // The Ethereum Web3 RPC URL, or the identifier of the network (e.g. `mainnet` or `goerli`)
+      zksync: true,
+      // Verification endpoint for Goerli
+      verifyURL: 'https://zksync2-testnet-explorer.zksync.dev/contract_verification'
+    }
 },
 // defaultNetwork: "zkTestnet", // optional (if not set, use '--network zkTestnet')
 ```
 
 Additional network properties:
 
-- `zkTestnet` is an arbitrary zkSync network name. You can select this as the default network using the `defaultNetwork` property.
-- `ethNetwork` is a field with the URL of the Ethereum node. You can also provide the network name (e.g. `goerli`) as the value of this field. In this case, the plugin will either use the URL of the appropriate Ethereum network configuration (from the `networks` section), or the default `ethers` provider for the network if the configuration is not provided. This field is required for all zkSync networks used by this plugin.
-- `url` is a field with the URL of the zkSync node in the case of the zkSync network (with `zksync` flag set to `true`), or the URL of the Ethereum node. This field is required for all zkSync and Ethereum networks used by this plugin.
-- `zksync` is a flag to indicate if the network represents zkSync network configuration. This field needs to be set to `true` for all zkSync networks. If you want to run a `hardhat-etherscan` verification, this field needs to be set to `false`. If set to `true`, the verification process will always try to run the verification process on the zkSync network.
+- `zkTestnet` is an arbitrary zkSync Era network name. You can select this as the default network using the `defaultNetwork` property.
+- `url` is a field with the URL of the zkSync Era node in case of the zkSync Era network (with `zksync` flag set to `true`), or the URL of the Ethereum node. This field is required for all zkSync Era and Ethereum networks used by this plugin.
+- `ethNetwork` is a field with the URL of the Ethereum node. You can also provide network name (e.g. `goerli`) as the value of this field. In this case, the plugin will either use the URL of the appropriate Ethereum network configuration (from the `networks` section), or the default. `ethers` provider for the network if the configuration is not provided. This field is required for all zkSync networks used by this plugin.
```

Comment:
```suggestion
- `ethNetwork` is a field with the URL of the Ethereum node. You can also provide network name (e.g. `goerli`) as the value of this field. In this case, the plugin will either use the URL of the appropriate Ethereum network configuration (from the `networks` section), or the default `ethers` provider for the network if the configuration is not provided. This field is required for all zkSync networks used by this plugin.
```

Reply:
I modified these descriptions in another section, I just copied/pasted into this section, does the order matter?

Reply:
it may well matter yes